build/proof-params/parameters.go

```diff
@@ -0,0 +1,19 @@
+package build
```

Comment:
why does this get moved, and why still `build`?

Reply:
These are needed by Curio, but Curio should NOT depend on Lotus's build/ folder which includes a large list of other things.

Reply:
ahh, is this to:

> Enables access to the latest params without depending on lotus/build

?

seems reasonable, but probably needs a new package name

Reply:
The naming is an oops. Fixed!